### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by aleh-rymasheuski**

### CODE REVIEW

File: src/ggrc_basic_permissions/__init__.py [hunk 1]
```diff
@@ -588,21 +588,19 @@ def context_relationship_query(contexts):
...
-      case([
-          (_relationship.destination_type == _context.related_object_type,
-           _relationship.source_type.label('type'))
-      ], else_=_relationship.destination_type.label('type')),
-      literal(None))
-
-  return db.session.query(*headers).join(_context, and_(
+  return db.session.query(
+      _relationship.source_id.label("id"),
+      _relationship.source_type.label("type"),
```
Comment (ahetmanski) [inline src/ggrc_basic_permissions/__init__.py:593]: First question: how are you?

File: src/ggrc_basic_permissions/__init__.py [hunk 2]
```diff
@@ -588,21 +588,19 @@ def context_relationship_query(contexts):
...
+      literal(None),
+  ).join(_context, and_(
       _context.id.in_(contexts),
       _relationship.destination_id == _context.related_object_id,
       _relationship.destination_type == _context.related_object_type,
-  )).union(db.session.query(*headers).join(_context, and_(
+  )).union(db.session.query(
+      _relationship.destination_id.label("id"),
+      _relationship.destination_type.label("type"),
```
Comment (ahetmanski) [inline src/ggrc_basic_permissions/__init__.py:601]: Are you still right?

File: src/ggrc_basic_permissions/__init__.py (hunk 1 above)
Comment (aleh-rymasheuski) [reply src/ggrc_basic_permissions/__init__.py:593]: Hi!

File: src/ggrc_basic_permissions/__init__.py (hunk 2 above)
Comment (aleh-rymasheuski) [reply src/ggrc_basic_permissions/__init__.py:601]: Testing answers in a separate review

**Review by aleh-rymasheuski — COMMENTED** [review]
Hello world!

**Comment by aleh-rymasheuski** [comment]
Done testing.

**Pull request closed without merging**